NuGet.config

```diff
@@ -3,6 +3,7 @@
      Licensed under the MIT License. -->
 <configuration>
     <packageSources>
+        <clear />
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
         <add key="ApplicationInsights-MyGet" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
```

Comment:
We pull the packages from this MyGet for testing, I could consider rehosting them in an upstream feed at ClrInstrumentationEngine-Dependency which is what our Unix build is using.